src/javascripts/modules/pillbar.js

```diff
@@ -0,0 +1,28 @@
+export default class Pillbar {
+  constructor(el) {
+    this.el = el
+    this.ticking = false
+    this.lastScrollY = 0
+
+    window.addEventListener('scroll', () => this.handleScroll())
```

Comment:
probably best to take some of these things out of the constructor and be deliberate about what you are doing.

a `setVars` method should have all your variables and `setupListeners` for your event listeners. it's best to do it this way just so you are sure sure to be explicit about what each method is doing so its clear to someone else coming on how it is organized without unneeded comments